### PR TITLE
Code of conduct updates dec2020

### PR DESCRIPTION
**Pull request by aburrell**
Merging updated code of conduct into develop.  This will only show the changes in the community documents after we've merged develop into main, but it will allow us to make the final changes to the document links and date beforehand.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+We at ELSP have chosen to adopt and follow the standards outlined in the
+Contributor Covenant Code of Conduct written for the development and use for
+Open Source Projects, at
+[Version 2.0](https://www.contributor-covenant.org/version/2/0/code_of_conduct/).
+This is version 1.0 of our Code of Conduct (ADD DATE AFTER ADDING LINKS).
+
+
+## Our Pledge
+
+We as editors, reviewers, and writers pledge to follow in the name of our
```
Comment (kzawdie) [inline CODE_OF_CONDUCT.md:12]: It says "pledge to follow", should this be "pledge to follow this code of conduct"

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+reported to the editorial board at equitable.space.letters@gmail.com.  All
+complaints will be reviewed and investigated promptly and fairly.
+
+All community leaders are obligated to respect the privacy and security of the
+reporter of any incident.
+
+### Enforcement Guidelines
+
+Here, ELSP will follow the Covenant Code of Conduct in determining the
+consequences any action(s) deemed to be a violation of this Code of Conduct:
```
Comment (kzawdie) [inline CODE_OF_CONDUCT.md:78]: Should be "consequences of any action(s)"

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+sustained inappropriate behaviour.
+
+Consequence: Barred from all community spaces (including acting as a reviewer or
+submitting a letter for review) for a specified period of time.  No public or
+private interaction with involved parties, including unsolicited interaction
+with those enforcing the Code of Conduct, should be initiated during this
+period.  Violating these terms may lead to a permanent ban.  If the temporary
+ban occurs in response to actions against an individual, the editorial board
+will offer to back the offended party should they wish to take further action,
+including notifying the offenders' institution or professional organisations.
```
Comment (kzawdie) [inline CODE_OF_CONDUCT.md:111]: Replace "organisations" with "organizations"

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+4.) Permanent Ban
+
+Community Impact: Demonstrating a pattern of violation of community standards,
+including sustained inappropriate behaviour, harassment of an individual, or
+aggression toward or disparagement of classes of individuals.
+
+Consequence: A permanent ban from the community.  May not act as a reviewer or
+editor, may not submit letters for review, may not participate in online or
+offline spaces.  The editorial board will contact the offenders' institution
+and relevant professional organisations if the behaviour also violates the
```
Comment (kzawdie) [inline CODE_OF_CONDUCT.md:122]: "Replace "organisations" with "organizations"

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+Code of Conduct of those communities.  If the permanent ban occurs in response
+to actions against an individual, the editorial board will offer to back the
+offended party should they wish to take further action.
+
+## Attribution
+
+This code of Conduct is adapted from the
```
Comment (kzawdie) [inline CODE_OF_CONDUCT.md:129]: Replace "code" with "Code"

File: CODE_OF_CONDUCT.md [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+- Not adhering to the guidelines for submitters and reviews (LINK HERE).
+
+## Enforcement Responsibilities
+
+Original creators and the current editorial board volunteers at ELSP are
+responsible for elucidating and enforcing standards of acceptable and
+unacceptable behavior.
+
+In the case of unacceptable behavior, ELSP, has the right to refuse to review
+and or provide feedback to contributors whose actions are not aligned with this
```
Comment (kzawdie) [inline CODE_OF_CONDUCT.md:54]: Should it be "and/or" instead of "and or"?

**Review by kzawdie — CHANGES_REQUESTED** [review]
Looks good, just a few small typo-type suggestions.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (aburrell) [reply CODE_OF_CONDUCT.md:12]:
```suggestion
We as editors, reviewers, and writers pledge to follow this code of conduct in the name of our
```

Comment (aburrell) [reply CODE_OF_CONDUCT.md:12]: I'm using the little (+/-) button to make suggestions that can be committed directly from here.

File: CODE_OF_CONDUCT.md (hunk 6 above)
Comment (aburrell) [reply CODE_OF_CONDUCT.md:54]:
```suggestion
and/or provide feedback to contributors whose actions are not aligned with this
```

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (aburrell) [reply CODE_OF_CONDUCT.md:78]:
```suggestion
consequences of any action(s) deemed to be a violation of this Code of Conduct:
```

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (aburrell) [reply CODE_OF_CONDUCT.md:111]:
```suggestion
including notifying the offenders' institution or professional organizations.
```

File: CODE_OF_CONDUCT.md (hunk 4 above)
Comment (aburrell) [reply CODE_OF_CONDUCT.md:122]:
```suggestion
and relevant professional organizations if the behavior also violates the
```

File: CODE_OF_CONDUCT.md (hunk 5 above)
Comment (aburrell) [reply CODE_OF_CONDUCT.md:129]:
```suggestion
This Code of Conduct is adapted from the
```

**Review by aburrell — COMMENTED** [review]
Using the "suggestion" button to make @kzawdie's suggestions.

**Review by kzawdie — APPROVED** [review]

**Pull request merged**